templates/repo/issue/milestone_new.tmpl

```diff
@@ -47,6 +47,15 @@
 						<a class="ui blue basic button" href="{{.RepoLink}}/milestones">
 							{{.i18n.Tr "repo.milestones.cancel"}}
 						</a>
+						{{if .IsClosed}}
+							<a class="ui button" href="{{.RepoLink}}/milestones/{{.MilestoneID}}/open">
```

Comment:
Are these done by GET requests? If so they should be changed to POST

Reply:
Pretty sure a `<a>` can not POST, so it either needs a `<form>` or some JS.

Reply:
Many thanks for pointing this out!

The new button is currently placed inside another `<form>` environment. So I guess it comes down either to relocate the button or to use JS?

Reply:
Yes, form nesting is forbidden, so I think it has to be done in JS.

Reply:
I have another idea: What about placing a checkbox inside the existing form?
If checked/unchecked it can flip the state of `.IfClosed` using the exsting modify button.

Reply:
Okay, I gave it a try with the checkbox. The code in this discussion is now obsolete.